src/item.cpp

```diff
@@ -1744,7 +1766,7 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
     }
 
     if (has_flag("FIT")) {
-        ret << _(" (fits)");
+        ret << _(" (fits) ");
```

Comment:
Oh yea, that's where my prefixes used to be. Looks like I didn't put everything back like I found it like I thought I did. Fixin'